polaris-react/src/components/ActionList/components/Item/Item.tsx

```diff
@@ -61,7 +62,16 @@ export function Item({
     );
   }
 
-  const contentText = ellipsis && content ? `${content}…` : content;
+  let contentText: string | React.ReactNode = content || '';
+  if (ellipsis) {
+    contentText = `${content}…`;
+  } else if (truncate && content) {
```

Comment:
nit: since we're deprecating `ellipsis` I think `truncate` should take precedence over it.